addonscript/script.py

```diff
@@ -0,0 +1,73 @@
+import aiohttp
+import asyncio
+import os
+
+import scriptworker.client
+
+from addonscript.api import do_upload, get_upload_status, get_signed_xpi
+from addonscript.task import build_filelist
+from addonscript.xpi import get_langpack_info
+
+
+def _craft_aiohttp_connector(context):
+    return aiohttp.TCPConnector()
+
+
+def get_default_config(base_dir=None):
+    base_dir = base_dir or os.path.dirname(os.getcwd())
+    default_config = {
+        'work_dir': os.path.join(base_dir, 'work_dir'),
+        'schema_file': os.path.join(os.path.dirname(__file__), 'data', 'addonscript_task_schema.json'),
+    }
+    return default_config
+
+
+async def sign_addon(context, locale):
+    # Retry Me
```

Comment:
Are these TODO markers?

We have `retry_async` in scriptworker that allows for retries. It could use improvement, but it's there and shared with all the *scripts if you want to use it.